### PR TITLE
Add banner for KubeCon 2024 EU

### PR DESCRIPTION
**Pull request by steve-hardman**
I have added banner for KubeCon + CloudNativeCon EU 2024 as requested in https://github.com/kubernetes/website/issues/44827

The banner looks like below

kubecon--banner

This will resolve https://github.com/kubernetes/website/issues/44827

### CODE REVIEW

File: data/announcements/scheduled.yaml
Comment (dipesh-rawat) [inline data/announcements/scheduled.yaml:221]: Year needs updating on this line.

```suggestion
- name: Kubecon 2024 EU
```

File: data/announcements/scheduled.yaml
Comment (dipesh-rawat) [inline data/announcements/scheduled.yaml:1]: @steve-hardman It would be greatly appreciated if you could incorporate a screenshot showcasing the banner in the pull request [description](https://github.com/kubernetes/website/pull/44828#issue-2092313939). It would enhance clarity and provide reviewers with a visual reference.

**Comment by dipesh-rawat** [comment]
/committee steering

This change will need steering committee approval.

File: data/announcements/scheduled.yaml
Comment (nate-double-u) [inline data/announcements/scheduled.yaml:225]: @cjyabraham, do you have the colours we should be using for this event?

File: data/announcements/scheduled.yaml
Comment (nate-double-u) [inline data/announcements/scheduled.yaml:223]: End date should be March
```suggestion
  endTime: 2024-03-22T18:00:00
```

File: data/announcements/scheduled.yaml
Comment (steve-hardman) [reply data/announcements/scheduled.yaml:221]: Thanks for the review. I will add them and include a screenshort too as requrested.

**Comment by cjyabraham** [comment]
Could we try using this banner ad:

kceu24_promo_2400x300


And the link should go here:
`https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/?utm_source=kubernetes&utm_medium=all&utm_campaign=kubeconeu24&utm_content=slim-banner`

**Comment by nate-double-u** [comment]
> https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/?utm_source=kubernetes&utm_medium=all&utm_campaign=kubeconeu24&utm_content=slim-banner

Even if we don't use this banner, we should use this link

**Comment by sftim** [comment]
> Could we try using this banner ad:
> 
> kceu24_promo_2400x300

We don't have support for banner advertisements that are images.

**Comment by steve-hardman** [comment]
> Even if we don't use this banner, we should use this link
I have changed the link to use the new link.

**Comment by steve-hardman** [comment]
One more thing, what dates should I use for the banner? For now I kept to start from 16 Feb i.e. 5 weeks before KubeCon.

File: data/announcements/scheduled.yaml
Comment (cjyabraham) [inline data/announcements/scheduled.yaml:228]: These needs to be the 2024 logo, not 2023.

**Review by cjyabraham — CHANGES_REQUESTED** [review]

File: data/announcements/scheduled.yaml
Comment (cjyabraham) [reply data/announcements/scheduled.yaml:225]: The ones he picked look good to me.

File: data/announcements/scheduled.yaml
Comment (steve-hardman) [reply data/announcements/scheduled.yaml]: Updated the image source to point to 2024 logo.

Comment (cjyabraham) [reply data/announcements/scheduled.yaml]: Thanks. Is it possible to make the Kubecon logo closer to the paragraph text on the left? And make the logo a bit bigger?

Comment (sftim) [reply data/announcements/scheduled.yaml]: The mechanism we have for banners only has basic support for images (they weren't something I'd planned to support, they just happen to kind-of work).

If you want that fine-grained control over how announcements appear @cjyabraham, you should file a separate feature request. Something I don't want to see happening: very custom CSS within the announcements data file.
- It's not a good precedent to set
- Every change there needs steering committee review, and they probably don't want to be involved in the minutiae of styling.

This PR is delivering what the associated issue says it should do.

---

PS. once the issue is filed, delivering the finer-grained control over appearance could take anything from days to years, depending on whether it gets worked on.

**Comment by cjyabraham** [comment]
Here's a suggested revision of the copy to get the date the correct format for Europe. I also removed "Virtual" as that doesn't seem to be a component of this year's event that is being pushed:

**Kubecon + CloudNativeCon Europe 2024**
4 days of incredible opportunities to collaborate, learn and share with the cloud native community.
19 - 22 March | Paris, France

@thetwopct any thoughts?

**Comment by nate-double-u** [comment]
> One more thing, what dates should I use for the banner? For now I kept to start from 16 Feb i.e. 5 weeks before KubeCon.

Thanks, I think those dates are good, that's what we had discussed earlier. Ultimately it'll be up to the steering committee.

**Comment by pohly** [comment]
:+1: from (steering committee member)

No strong opinion about the timing.

File: data/announcements/scheduled.yaml
Comment (stmcginnis) [inline data/announcements/scheduled.yaml:229]: Link is still pointing to the North America event. Can you update to the EU landing page?

```suggestion
    <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/">Kubecon + CloudNativeCon Europe 2024</a>
```

**Review by cjyabraham — APPROVED** [review]

File: data/announcements/scheduled.yaml
Comment (natalisucks) [inline data/announcements/scheduled.yaml:221]:
```suggestion
- name: KubeCon 2024 EU
```

File: data/announcements/scheduled.yaml
Comment (natalisucks) [inline data/announcements/scheduled.yaml:229]:
```suggestion
    <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/">KubeCon + CloudNativeCon Europe 2024</a>
```

File: data/announcements/scheduled.yaml
Comment (natalisucks) [inline data/announcements/scheduled.yaml:231]:
```suggestion
    Join us for four days of incredible opportunities to collaborate, learn and share with the cloud native community.<br/> 
```

**Review by natalisucks — DISMISSED** [review]
@cjyabraham please note that only approvers of the Kubernetes website can approve PRs going upstream

@steve-hardman thanks for this PR and working with folks on feedback here – i've requested changes due to the name of the event needing to be correctly capitalized in the banner, alongside our/the event's style guide around small numbers and writing them out

I'll be checking if two members of steering giving their OK for these changes is enough for us to approve, but if anyone from steering can confirm, that would be great

**Comment by natalisucks** [comment]
/hold
holding this work until the changes requested are adopted and we have steering majority before merging

**Comment by mrbobbytables** [comment]
lgtm from me with steering hat on for the general msg 👍

**Comment by pacoxu** [comment]
+1 with steering hat

File: data/announcements/scheduled.yaml
Comment (steve-hardman) [reply data/announcements/scheduled.yaml:229]: Sorry! I used the existing lines as reference and hence the mistake. I accpeted the suggested coorections. 
After this commit it means new lines from this PR will be different from all the past lines; hope that's okay!

**Comment by sftim** [comment]
Thanks. If you know how to squash down to 1 commit, please do that (it's tidier). It's OK if you don't know how to - Git is definitely not easy to learn.

**Comment by sftim** [comment]
This is OK to unhold as soon as it is fully approved. That's because Steering will only provide formal approval once they have a majority consensus for approval.

**Comment by steve-hardman** [comment]
> If you know how to squash down to 1 commit, please do that (it's tidier).

Thanks for review! Done completed the squash, I learnt how to do this based on steps shared by @dipesh-rawat in one of my PR earlier last year.

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

**Comment by natalisucks** [comment]
/remove-hold
/approve

Thanks @BenTheElder for clarification of the majority met from Steering, and thanks once again to @steve-hardman and all reviewers

**Comment by nate-double-u** [comment]
Thanks everyone!

File: data/announcements/scheduled.yaml
Comment (cjyabraham) [inline data/announcements/scheduled.yaml:229]: Could we get the utm querystring arguments on this link as well please? So make the link:
`https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_source=kubernetes&utm_medium=all&utm_campaign=kubeconeu24&utm_content=slim-banner`

**Pull request merged**
